test/integrations/destinations/ga4/dataDelivery/data.ts

```diff
@@ -1,177 +1,3 @@
-export const data = [
```

Comment:
Lets not delete test cases for `v0`  along with these lets add `testScenariosV1API`

Reply:
addressed